docs/README.md

```diff
@@ -128,8 +128,40 @@ For more information on how to use the `objdef` command, see [Supporting extensi
 
 Sub-commands:
 
-* `validate` (or `v`): Validate a scenario script.
-* `template` (or `tmpl` or `t`): Generate a template scenario script.
+* `connect`   (or `c`): Test connection using settings provided by the config file.
+* `structure` (or `s`): Get app structure using settings provided by the config file.
```

Comment:
Change to:
"Get the app structure using the settings provided in the config file."